v0/destinations/af/transform.js

```diff
@@ -186,17 +185,12 @@ function processEventTypeTrack(message) {
       break;
     }
   }
-  let payload;
-  if (isUnIdentifiedEvent) {
```

Comment:
This variable is unused. Is that intentional? Let's clean up the other unused code if that's the case.

Reply:
Yes, it was useful as a check earlier, if the event was not belonging to one the declared categories of events with own mappings. Unused so removing it now.